core/controllers/profile.py

```diff
@@ -361,6 +364,7 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    @acl_decorators.open_access
```

Comment:
the request only goes from logged in users [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/i18n.js#L89) ... I think I should just remove the test?